met-api/src/met_api/utils/template.py

```diff
@@ -1,8 +1,10 @@
 """Template Services."""
 
+import os
 from jinja2 import Environment, FileSystemLoader
 
-ENV = Environment(loader=FileSystemLoader('./templates'), autoescape=True)
+templates_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '../../..', 'templates'))
```

Comment:
@saravanpa-aot this is back, when using the package (like cron does) it won't find the template again, this is an attempt fix, i'm not sure if it solves the problem yet

Reply:
hm..do we need this if we are doing gc notify?